app/org/sagebionetworks/bridge/models/accounts/ParticipantOptions.java

```diff
@@ -0,0 +1,61 @@
+package org.sagebionetworks.bridge.models.accounts;
+
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+import javax.annotation.Nullable;
+
+import org.sagebionetworks.bridge.dao.ParticipantOption.SharingScope;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Object to deserialize JSON update of participant options. Some or all of the 
+ * values can be missing in the JSON and we will only update values that are non-null.
+ */
+public final class ParticipantOptions {
+
+    private final @Nullable SharingScope sharingScope;
+    private final @Nullable Boolean notifyByEmail;
+    private final @Nullable String externalId;
+    private final @Nullable Set<String> dataGroups;
+    private final @Nullable LinkedHashSet<String> languages;
```

Comment:
Discussed this offline with Alx. The main concern here is that we have a quadratically growing list of things to change when we update ParticipantOptions, which makes this a maintainability nightmare.

On the flip side, we need to be able to parse the incoming request based on specific options and types. I think the best way to handle this would be to have the enum itself contain the parsing logic. This way, it's easy to maintain, type-safe, and the ParticipantOptions.java could still just be a map.